core/sql/regress/compGeneral/EXPECTED042

```diff
@@ -1871,8 +1871,8 @@ NUM_ENTRIES  TEXT_ENTRIES  NUM_PLANS
 NUM_ENTRIES  TEXT_ENTRIES  NUM_PLANS 
 -----------  ------------  ----------
 
-         17             0          17
-          1             0           1
+         22             0          22
```

Comment:
Does this mean more metadata queries are being cached now? Dow we know what the new entries are? There is no doubt that the fix for 1494 is necessary and required. I am not able to understand fully the changes in this expected file. If it will take too much effort to find out, we could simply accept the result. I am good with that too.